src/Jackett.Common/Indexers/BakaBT.cs

```diff
@@ -128,7 +128,7 @@ private async Task DoLogin()
             {
                 var parser = new HtmlParser();
                 var dom = parser.ParseDocument(responseContent);
-                var messageEl = dom.QuerySelectorAll(".error").First();
+                var messageEl = dom.QuerySelectorAll("#loginError").First();
                 var errorMessage = messageEl.Text().Trim();
```

Comment:
where did you get `#loginError` from ?
when a login fails I see 
```
                <p class="error">
                    Username or password is incorrect. You still have <b>5</b>
                    attempt(s)
                </p>
``` 
so I would expect that `p.error` would be the correct detector?

Reply:
Never mind.
I see what you found when using the /login.php page instead of the default domain page that you get if you access the site without a previous logged in session.